pre/NSIDC-OSTIA_SST-ICE_blend/cube_BCs.pl

```diff
@@ -159,7 +159,7 @@ sub write_and_submit_jobfile {
     if ( $npn == 40 ) {
       $mynodes = "sky";
     } else {
-      $mynodes = "hasw";
+      $mynodes = "sky|cas";
```

Comment:
Just making sure...did you test this? I only ask because I know SLURM/shells are very annoying with it. (Sometimes I seem to need quote or brackets or...)

Reply:
@mathomp4 It worked for me

Reply:
That's all I care about! Thanks for testing it! 😄